airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -34,8 +34,8 @@ tests:
       future_state_path: "integration_tests/abnormal_state.json"
       cursor_paths:
         subscription_changes: ["timestamp"]
-        email_events: ["timestamp"]
-        contact_lists: ["timestamp"]
+        email_events: ["created"]
```

Comment:
if it matches the cursor_field in a stream (catalog) then you don't need cusor_paths for this stream (and any of them)
the purpose of `cursor_paths` is to handle cases when the path in the STATE file is nested
```json
"some_field": {
   "cursor_field": "value"
}
```